packages/fast-components-react-base/src/select/select.tsx

```diff
@@ -582,6 +639,13 @@ class Select extends Foundation<SelectHandledProps, SelectUnhandledProps, Select
             triggerButton.focus();
         }
     };
+
+    /**
+     * get valid options
+     */
+    private getValidOptions = (): React.ReactNode[] => {
```

Comment:
yep

Reply:
Actually let's not pull on that string right now.  That comes from  the getValidOptions() function which calls "React.Children.toArray" which has"React.ReactNode[]" as a return type that we're using and that extends into lisbox as well.  I don't think that because React.ReactNode can be an array doesn't mean that React.ReactNode[] isn't also a valid construct.

Reply:
It's not that it isn't a valid construct it's just that it's unnecessary since it's already taken care of. You can see it in the definition here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v15/index.d.ts#L186

Where it can be a ReactNodeFragment which itself can be an array of ReactNode.

Not blocking on it though.